framework/src/postprocessors/ElementExtremeValue.C

```diff
@@ -31,16 +31,21 @@ ElementExtremeValue::validParams()
                              "returns the maximum value. 'min' returns "
                              "the minimum value.");
 
+  params.addCoupledVar("proxy_variable",
```

Comment:
Should probably add a comment here describing what happens when this isn't provided. same for the nodal object